### PR TITLE
add POVR.yml

### PR DESCRIPTION
**Pull request by JaseNZC**
Request from #73 (end of thread)

Covers all 18 studios on the site.

I have not removed WankzVR or MIlfVR as they are sites on their own.

Date is a bit Janky but does work, Change it if you want a better way to pull it :-)

### CODE REVIEW

File: scrapers/POVR.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    scene:
+      Title:
+        selector: //span[@class="video__quality video__quality--vr mr-3"]/following-sibling::text()
+      Date:
+        selector: //p[contains(@class, 'player__date')]
+        postProcess:
+          - replace:
+              - regex: ^.{1,11}(.*)$
+                with: $1
+          - parseDate: 02 January, 2006
```
Comment (bnkai) [inline scrapers/POVR.yml:18]: for the date imo it is better to use a regex bassed on content than positioning
the date in the site does not include 0 padding btw
```yaml
      Date:
        selector: //p[contains(@class, 'player__date')]
        postProcess:
          - replace:
              - regex: .*(\d+\s[a-zA-Z]+,\s\d+)$
                with: $1
          - parseDate: 2 January, 2006
```

Comment (JaseNZC) [reply scrapers/POVR.yml]: Done, Told ya I had a Janky date there lol.

Comment (bnkai) [reply scrapers/POVR.yml]: @JaseNZC i added a last second  \s to the regex before the (
(your merge was faster than my edit :-) )

Comment (JaseNZC) [reply scrapers/POVR.yml]: Bam, Lightning fast.

**Pull request merged**